pkg/machine/hyperv/machine.go

```diff
@@ -444,7 +449,7 @@ func (m *HyperVMachine) SSH(name string, opts machine.SSHOptions) error {
 	if username == "" {
 		username = m.RemoteUsername
 	}
-	return machine.CommonSSH(username, m.IdentityPath, m.Name, m.Port, opts.Args)
+	return machine.CommonSSH("core", m.IdentityPath, m.Name, m.Port, opts.Args)
```

Comment:
I think this line makes the previous 3 lines a noop?

Reply:
yeah, that slipped in.  ill fix it